tracegen/internal/tracegen/config.go

```diff
@@ -78,6 +79,11 @@ func (c *Config) Flags(fs *flag.FlagSet) {
 	c.Headers = make(map[string]string)
 	fs.Var(&c.Headers, "otlp-header", "Custom header to be passed along with each OTLP request. The value is expected in the format key=value."+
 		"Flag may be repeated to set multiple headers (e.g -otlp-header key1=value1 -otlp-header key2=value2)")
+
+	// custom resource attributes
+	c.ResourceAttributes = make(map[string]string)
```

Comment:
Resource attributes can have other types like e.g. integers or booleans. I think it's fine to only add support for strings first, but we should think about how we will add support for other types in the future. 

To make it concrete: right now `-otlp-attributes key=true` will create an attribute with the string value `"true"`, which can be confusing. Should we force people to add quotes to wrap the values? (so .e.g. you have to do `key="true"` for adding a string vs `key=true` for adding a literal boolean?)

Reply:
Actually maybe this gets solved if you do `map[string]interface{}` instead of `map[string]string`?

Reply:
It's fine to not support these now, but we need to be able to detect them now. My suggestion is to do the following:

```
func (v *KeyValue) Set(s string) error {
	kv := strings.SplitN(s, "=", 2)
	if len(kv) != 2 {
		return fmt.Errorf("value should be of the format key=\"value\"")
	}
        val := kv[1]
        if len(val) < 2 || val[0] != "\"" || val[len(val) - 1] != "\" {
                return fmt.Errorf("value should be a string wrapped in double quotes")
        }

	(*v)[kv[0]] = val[val[1 : len(val)-1]]
	return nil
}
```

In this way, `key="true"` will pass (setting to the string value `"true"`) while `key=true` will fail (and we can support it later).

Reply:
Thanks for your review, after adding this logic:
```bash
-otlp-insecure -duration 5s -otlp-attributes key1=\"value2\" -otlp-attributes key2=\"123\" -otlp-attributes key3=\"true\" 
```